test/util.mjs

```diff
@@ -11,12 +34,21 @@ export function compareDir(dir, newDir = `${dir}-new`, extensions = ['png', 'bin
 
 export function readFile(file) {
 	try {
-		return readFileSync(file);
+		return readFileSync(file).toString();
 	} catch (e) {
 		return null;
 	}
 }
 
+function compareMarkers(markerA, markerB) {
```

Comment:
WDYT about `compareMarkers` -> `isEqualMarker`?

Reply:
Yup, that's better